Modules/SceneSerialization/src/mitkPointSetSerializer.cpp

```diff
@@ -37,6 +37,12 @@ std::string mitk::PointSetSerializer::Serialize()
     return "";
   }
 
+  // do not save empty PointSet
+  if (ps->GetSize() == 0)
```

Comment:
Please allow me adding my thoughts on this one. With this change applied, do we still get the DataNode around this PointSet correctly restored? This could be important conserve the hierarchy of DataStorage unmodified. I would discourage skipping certain data objects due to considerations about their usefulness (their properties or their node's properties could still be significant for the application).

Reply:
Agreed, I actually need to re-test if problems with saving empty point set are still being actual or not, will close this PR for now